src/coreclr/gc/gcpriv.h

```diff
@@ -1154,6 +1155,41 @@ enum interesting_data_point
     max_idp_count
 };
 
+#ifdef USE_REGIONS
+enum free_region_kind
+{
+    basic_free_region,
+    large_free_region,
+    huge_free_region,
+    count_free_region_kinds,
+};
+
+class region_free_list
+{
+    size_t  num_free_regions;
+    size_t  size_free_regions;
+    size_t  size_committed_in_free_regions;
+    size_t  num_free_regions_added;
+    size_t  num_free_regions_removed;
+    heap_segment* head_free_region;
+    heap_segment* tail_free_region;
+
```

Comment:
nit - seeing that these are all for free regions any way? is it better to not have free_regions in the name of every member? could be `count`, `size`, `committed`, `added`, `removed`, `head`, `tail`?

Reply:
Well, later searching for places where these fields are used is easier with the more verbose names. So, I'd rather keep them...